docs/developer-docs/functionality/sns/index.md

```diff
@@ -0,0 +1,74 @@
+# How to launch an SNS
+
+We explain how to deploy an SNS that is provided as a 
+system functionality by the IC.
+As explained in the [introduction](sns.md), there are also other 
```

Comment:
Does this mean there are two introductory/index-like pages that should be merged? ('No' is a complete answer, too)

Reply:
No :) One page introduced the SNS (the one that is linked) and this page here is actually an index to "how to launch an SNS" that then has multiple sub-pages.. 
If there is a better way to reflect that in the structure, please feel free to suggest something!

Reply:
renamed the file and will resolve this thread